package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@remoteoss/json-schema-form",
-  "version": "0.9.1-beta.0",
+  "version": "0.9.2-dev.20240518112325",
```

Comment:
Great, thanks! Do you have a MR in our internal codebase to ensure the pipelines are still green? Can you link it here?

Reply:
Yep, corresponding MR is [here](https://gitlab.com/remote-com/employ-starbase/dragon/-/merge_requests/26523).

Reply:
Perfect, then yes, let's go with this one. Revert the version with `npm run version_as_main` and we can merge this